machines/auth.ts

```diff
@@ -236,3 +247,10 @@ export function selectLanguagesetup(state: State) {
 export function selectIntroSlider(state: State) {
   return state.matches('introSlider');
 }
+
+export function selectIsBiometricToggleFromSettings(state: State) {
+  if (state.matches('settingUp')) {
```

Comment:
if machine is in `settingUp` state then toggleFromSettings should be false right? toggleFromSettings will be true if we come from settings screen and at that time we should not be in `settingUp` state

Reply:
Default value of toggleFromSettings is false. it will be set to true only when we toggle it from the settings.